app/clients/__init__.py

```diff
@@ -11,3 +11,28 @@ class Client(object):
     Base client for sending notifications.
     '''
     pass
+
+
+STATISTICS_REQUESTED = 'requested'
+STATISTICS_DELIVERED = 'delivered'
+STATISTICS_FAILURE = 'failure'
+
+
+class ClientResponse:
```

Comment:
Base response class.

- Used to map status codes from SMS/Email providers back into Notify statuses